internal/common/validator.go

```diff
@@ -17,8 +18,8 @@ const (
 
 type ValidationError struct {
 	Field string `json:"field"`
-	Rule  string `json:"rule"`
 	Value string `json:"value,omitempty"`
+	Tag   string `json:"tag"`
```

Comment:
If we keep that field (and I think we can keep it after all) it must be named `rule` as [per the API spec](https://github.com/italia/developers-italia-api/blob/main/developers-italia.oas.yaml#L1551-L1556).